torch/_prims/context.py

```diff
@@ -156,3 +162,42 @@ def __torch_function__(
                 return fn(*args, **kwargs)  # type: ignore[operator]
 
         return func(*args, **kwargs)
+
+
+@functools.lru_cache
+def torch_to_refs_map():
+    modules = [
+        (torch, torch._refs),
+        (torch.nn, torch._refs.nn),
+        (torch.nn.functional, torch._refs.nn.functional),
+        (torch.special, torch._refs.special),
+    ]
+    r = {}
+    for mod_torch, mod_refs in modules:
+        for s in mod_refs.__all__:
+            r[mod_torch.__dict__.get(s)] = mod_refs.__dict__.get(s)
+    return r
+
+class TorchRefsMode(torch.overrides.TorchFunctionMode):
+    """
+    Switches the interpretation of torch.* functions and Tensor methods to
+    use PrimTorch refs in torch._refs.
```

Comment:
One more sentence with a simple example would be nice, also note that direct calls to refs are preserved?